css/components/counter/counter.css

```
@@ -0,0 +1,25 @@
+.counter {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.counter__question,
+.counter__bookmark {
+  display: flex;
+  align-self: center;
+  margin-left: 10px;
+}
+
+.counter__question {
+  margin-right: 15px;
+}
+
+.counter__question--image {
+  height: 30px;
+}
+
+.counter__bookmark--image {
+  height: 30px;
+  margin-left: 15px;
+}
```

As i don't know BEM that much, I am unsure if you would write:
```suggestion
.counter__question__image {
  height: 30px;
}

.counter__bookmark__image {
  height: 30px;
  margin-left: 15px;
}
```

because the image is an element itself und not a modification of an object... but I am not sure about the writing :)